package.json

```diff
@@ -4,7 +4,10 @@
   "author": "Daniel Wirtz <dcode@dcode.io>",
   "description": "The swiss army knife for binary data in JavaScript.",
   "main": "index.js",
-  "browser": "dist/ByteBufferAB.js",
+  "browser": {
+    "index.js": "bytebuffer/dist/ByteBufferAB.js",
+    "Long": "bytebuffer/node_modules/long/dist/Long.js"
```

Comment:
This doesn't look browserify safe, also the Long value should not be in this module. It should be resolved by long's package.json.